ethereum/test/pyth.js

```diff
@@ -617,7 +497,18 @@ contract("Pyth", function () {
     await updatePriceFeeds(this.pythProxy, [rawBatch]);
 
     // Setting the validity time to 30 seconds
-    await this.pythProxy.updateValidTimePeriodSeconds(30);
+    await this.pythProxy.executeGovernanceInstruction(
```

Comment:
this could also be worthwhile to extract into a test helper function to avoid code duplication

Reply:
yeah I agree, I was a bit lazy :D will do that.